src/content/configuration/optimization.md

```diff
@@ -155,19 +155,19 @@ module.exports = {
 };
 ```
 
-## `optimization.noEmitOnErrors`
+## `optimization.emitOnErrors`
```

Comment:
Our docs contains information for webpack@4 and webpack@5? If yes I think we need note what previously this options had the `noEmitOnErrors` name

Reply:
Docs are v5, if something breaking is merged there is a v4 branch to do PRs against